### PR TITLE
[SYCL] Fix typo in the name of the host-visible pool

### PR DESCRIPTION
**Pull request by smaslov-intel**
Signed-off-by: Sergey V Maslov <sergey.v.maslov@intel.com>

### CODE REVIEW

File: sycl/plugins/level_zero/pi_level_zero.cpp [hunk 1]
```diff
@@ -4761,7 +4761,7 @@ pi_result piEventCreate(pi_context Context, pi_event *RetEvent) {
   if (ZeAllHostVisibleEvents) {
     ZeEventDesc.signal = ZE_EVENT_SCOPE_FLAG_HOST;
   } else {
-    ZeEventDesc.signal = ZE_EVENT_SCOPE_FLAG_DEVICE;
```
Comment (againull) [inline sycl/plugins/level_zero/pi_level_zero.cpp:4764]: I don't completely understand this change. So do you basically want to change this to ZE_EVENT_SCOPE_FLAG_SUBDEVICE?

Comment (smaslov-intel) [reply sycl/plugins/level_zero/pi_level_zero.cpp:4764]: No, the current recommendation from L0 GPU RT is that "0" scope should be used. The ZE_EVENT_SCOPE_FLAG_SUBDEVICE is "1".

Comment (againull) [reply sycl/plugins/level_zero/pi_level_zero.cpp:4764]: Hm, but per specification it looks like ZE_EVENT_SCOPE_FLAG_SUBDEVICE is "0" and ZE_EVENT_SCOPE_FLAG_DEVICE is "1".
https://spec.oneapi.io/level-zero/latest/core/api.html?highlight=ze_event_scope_flag_device#_CPPv426ZE_EVENT_SCOPE_FLAG_DEVICE

Comment (smaslov-intel) [reply sycl/plugins/level_zero/pi_level_zero.cpp:4764]: ZE_BIT(0) = 1 << 0 = 1

Comment (againull) [reply sycl/plugins/level_zero/pi_level_zero.cpp:4764]: Ok, sorry

**Review by againull — APPROVED** [review]

**Pull request merged**